.github/workflows/ci.yaml

```diff
@@ -4,6 +4,13 @@ on:
     branches: [main]
   pull_request: null # target every PR
 jobs:
+  allchecks:
+    needs: [test, build, fmt, lint, cargo-deny]
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Out of interest, did you try without any steps? I'm assuming GHA complains but I can't see where the docs say what keys are mandatory though...

Reply:
Yes I have triedit. `runs-on` and `steps` are mandatory statements and workflow fails without it

Reply:
Thanks for confirming...